README.md

```diff
@@ -25,6 +25,18 @@ To get code coverage information, you can run pytest directly.
 $ pytest --cov=src --cov-report=html
 ```
 
+# Pushing to staging
+The staging environment for this service is
+[here](https://selected-tests.server-tig.prod.corp.mongodb.com/swagger). In order to test your
```

Comment:
This should be https://selected-tests.server-tig.staging.corp.mongodb.com/swagger, I think?

Reply:
You're right, bad copy/paste job. Fixed now